precompile/fee_config_manager.go

```diff
@@ -56,6 +56,7 @@ var (
 type FeeConfigManagerConfig struct {
 	AllowListConfig // Config for the fee config manager allow list
 	UpgradeableConfig
+	InitialFeeConfig *commontype.FeeConfig `json:"initialFeeConfig,omitempty"` // initial fee config to be immediately activated
```

Comment:
Should we verify that this is valid on startup, so that if there's an invalid fee config then it will cause the node to refuse to start when it's first passed in as opposed to causing a panic when the upgrade activates?

Reply:
yeah good point

Reply:
done with Verify()